packages/shared/src/components/modals/common/ModalHeader.tsx

```diff
@@ -1,18 +1,27 @@
-import React, { ReactElement, ReactNode } from 'react';
+import React, { ReactElement, ReactNode, useContext } from 'react';
+import classed from '../../../lib/classed';
 import { ModalClose } from './ModalClose';
 import { ModalPropsContext } from './types';
 
 export type ModalHeaderProps = {
-  title?: ReactNode;
+  children?: ReactNode;
+  title?: string;
 };
 
-export function ModalHeader({ title }: ModalHeaderProps): ReactElement {
+const ModalHeaderTitle = classed('h3', 'font-bold typo-title3');
```

Comment:
I'm all for these, but why do we used it for the title, but not the header?
Perhaps we should find a uniform for this?

Reply:
I used for the header to be able to expose it outside of this component. Do we need it for everything?
My thought was to only use it for exposed components.